app/models/chat_channel.rb

```diff
@@ -99,6 +154,32 @@ def self.is_enabled?(t)
 
     ChatChannel.where(chatable: topic).exists?
   end
+
+  private
+
+  def log_channel_status_change(acting_user:, new_status:, old_status:)
+    new_status = new_status.to_sym
+    old_status = old_status.to_sym
```

Comment:
Do we need a default value for old_status?

Reply:
No, it will always have a value. The default status for a channel is `"open"`